changelog/fixed-target-gen-registry.md

```diff
@@ -0,0 +1 @@
+Fixed target-gen registry not getting used by probe-rs
```

Comment:
If it's not a bug that existed in the previous release, it doesn't need a changelog entry.

Reply:
I can remove the changelog patch. I kept it separate in case that was an issue, but I added the changelog because CI was failing.

Would you like me to remove the changelog entry and force-push?

Reply:
Could you maybe replace it with one that mentions the new API in Session? The force push doesn't matter, it'll be squashed anyway.

Reply:
Sure, I've done that. Thanks for the suggestion.

Reply:
Thanks!